features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/CallScreenPresenter.kt

```diff
@@ -121,6 +121,15 @@ class CallScreenPresenter @AssistedInject constructor(
 
         callWidgetDriver.value?.let { driver ->
             LaunchedEffect(Unit) {
+                loadCallTimeoutJob = launch {
```

Comment:
Maybe we could just use : 
```
withTimeout(10){
driver.incomingMessages.firstOrNull {parsedMessage.action == WidgetMessage.Action.Join }
}
```
This would be a bit less convoluted